testtools/tests/test_testcase.py

```diff
@@ -1967,6 +1967,39 @@ def test_other_attribute(self):
         self.assertRaises(AttributeError, getattr, orig, 'thing')
 
 
+class TestTCaseExpectThat(TestCase):
+
+    def setUp(self):
+        super(TestTCaseExpectThat, self).setUp()
+        self.passed = None
+        self.message = 'default'
+
+    def expectThat(self, matcher, matchee, message='', verbose=False):
```

Comment:
I don't think there is any need for this helper to shadow expectThat, and in shadowing it it will surprise folk - as its not a method one would expect to be shadowed.